AIR/donations.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- ticker: AIR
+  announcement_url: https://onetreeplanted.org
+  recipient:
+    name: Save the Children
+    address: 501 Kings Highway East, Suite 400, Fairfield, CT 06825
+    tel: +1 800 728 3843 
+    contact: supportercare@savechildren.org
+  ada: 258
+  usd: 30
+  timestamp: 1602460800
+  files:
+     - No files provided
+  charity_tx_id: Invoice No. 17130
```

Comment:
HI Kiriakos! 

I just added the files and did a commit. 

The second donation, all good. receipt, etc.

Unfortnaley, in the first donation I just have the print of transfer because, if I remenber correctly, I did an anonymous BTC donation so I did not received a receipt. But in the print there is the reference to the transference ID.

Thanks!